Makefile

```diff
@@ -61,20 +70,20 @@ production-image-push: production-image			## Push production images to Docker Hu
 	docker image push --all-tags ${DOCKER_IMAGE_WEBAPP}
 	docker image push --all-tags ${DOCKER_IMAGE_PROXY}
 
-production-deploy: production-image-push		## Deploy production to AWS
+production-deploy: production-image-push 		## Deploy production to AWS
 	scp \
 		-i .secrets/labyrinth-ed25519.pem \
 		docker-compose.yml docker-compose.prod.yml scripts/run-production.sh \
-		ubuntu@ec2-13-37-240-163.eu-west-3.compute.amazonaws.com:~
+		${SERVER_USER}@${SERVER_HOSTNAME}:~
 
 	ssh \
 	 	-i .secrets/labyrinth-ed25519.pem \
-	 	ubuntu@ec2-13-37-240-163.eu-west-3.compute.amazonaws.com \
+	 	${SERVER_USER}@${SERVER_HOSTNAME} \
 	 	'echo "TAG=${COMMIT_HASH}" > .env'
 	
 	ssh \
 	 	-i .secrets/labyrinth-ed25519.pem \
-	 	ubuntu@ec2-13-37-240-163.eu-west-3.compute.amazonaws.com \
+	 	${SERVER_USER}@${SERVER_HOSTNAME} \
 	 	'./run-production.sh'
 
 production: production-image					## Run the program for production
```

Comment:
Glad you changed `production-image` to `production-image-build` :slightly_smiling_face: 

However it seems you forgot to update the make target dependencies (this line, + at least 1 other line I believe) :grimacing: